platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -399,24 +260,19 @@ public class SwirldsPlatform implements Platform, ConnectionTracker, Startable {
     private final PreConsensusEventWriter preConsensusEventWriter;
 
     /**
-     * True if gossip has been halted. Only relevant for sync-as-a-protocol
-     */
-    private final AtomicBoolean gossipHalted = new AtomicBoolean(false);
-
-    /**
-     * The permit provider for syncs. Only relevant when sync is running as a protocol
+     * Responsible for transmitting and receiving events from the network.
      */
-    private SyncPermitProvider syncPermitProvider;
+    private final Gossip gossip;
 
     /**
      * the browser gives the Platform what app to run. There can be multiple Platforms on one computer.
      *
+     * @param platformContext          the context for this platform
      * @param crypto                   an object holding all the public/private key pairs and the CSPRNG state for this
+     * @param initialAddressBook       the address book listing all members in the community
```

Comment:
did this addition split the documentation for the `crypto` paramter?   The next line is `member` which I think belongs after the `this` in the previous line.

Reply:
yes, fixed